accessible/field-segment.component.js

```diff
@@ -144,7 +150,12 @@ blocklyApp.FieldSegmentComponent = ng.core.Component({
   // Confirm a selection for dropdown fields.
   selectOption: function() {
     if (this.optionValue == Blockly.Msg.RENAME_VARIABLE) {
```

Comment:
Yeah - I think both things are actually wrong - Comparing indices is prone to reordering, string comparison is prone to translation issues. We need a common code path with Web Blockly to figure out whether an option is rename or delete. Which I'll do for the next update. 👍